samples/openapi/main.go

```diff
@@ -87,7 +87,7 @@ func openAPIGetRawJsonAuditTrail(vcdClient *govcd.VCDClient) {
 	queryParams.Add("filter", "timestamp=gt="+filterTime)
 
 	allResponses := []json.RawMessage{{}}
-	err = vcdClient.Client.OpenApiGetAllItems("33.0", urlRef, queryParams, &allResponses, nil)
+	err = vcdClient.Client.OpenApiGetAllItems("34.0", urlRef, queryParams, &allResponses, nil)
```

Comment:
similar comment -  maybe we don't need this restriction. @Didainius

Reply:
We can set this. It would probably complain about it now.